src/containers/project.py

```diff
@@ -1,35 +1,210 @@
 from typing import List, Dict
+import requests
 
 from .project_issue import ProjectIssue
+from .base_container import BaseContainer
 
+ISSUE_PER_PAGE = 100
+ISSUES_FROM_PROJECT_QUERY = """
+    query {{
+      node(id: "{project_id}") {{
+        ... on ProjectV2 {{
+          items(first: {issues_per_page}, {after_argument}) {{
+            pageInfo {{
+              endCursor
+              hasNextPage
+            }}
+            nodes {{
+              content {{
+                  ... on Issue {{
+                    title
+                    state
+                    number
+                    repository {{
+                      name
+                      owner {{
+                        login
+                      }}
+                    }}
+                  }}
+                }}
+              fieldValues(first: 100) {{
+                nodes {{
+                  __typename
+                  ... on ProjectV2ItemFieldSingleSelectValue {{
+                    name
+                  }}
+                }}
+              }}
+            }}
+          }}
+        }}
+      }}
+    }}
+    """
+PROJECT_FIELD_OPTIONS_QUERY = """
+        query {{
+          repository(owner: "{org_name}", name: "{repo_name}") {{
+            projectV2(number: {project_number}) {{
+              title
+              fields(first: 100) {{
+                nodes {{
+                  ... on ProjectV2SingleSelectField {{
+                    name
+                    options {{
+                      name
+                    }}
+                  }}
+                }}
+              }}
+            }}
+          }}
+        }}
+        """
 
-class Project:
+
+class Project(BaseContainer):
     def __init__(self):
         self.id: str = ""
         self.number: int = 0
         self.title: str = ""
-        self.organizationName: str = ""
-        self.repositoriesFromConfig: List[str] = []
-        self.projectRepositories: List[str] = []
+        self.organization_name: str = ""
+        self.config_repositories: List[str] = []
+        self.project_repositories: List[str] = []
         self.issues: List[ProjectIssue] = []
-        self.fieldOptions: Dict[str, List[str]] = {}
+        self.field_options: Dict[str, List[str]] = {}
+
+    def to_dict(self):
+        return {
+            'id': self.id,
+            'number': self.number,
+            'title': self.title,
+            'organization_name': self.organization_name,
+            'config_repositories': self.config_repositories,
+            'project_repositories': self.project_repositories,
+            'issues': self.issues,
+            'field_options': self.field_options
+        }
+
+    def load_from_api_json(self, repository, gh_project):
```

Comment:
This method does not need to load from json string/dict.

Reply:
Method changed, so it takes GHProject object. No need for converting into dict.